base/array.jl

```diff
@@ -27,11 +27,12 @@ import Core: arraysize, arrayset, arrayref
 
 """
     Array{T}(dims)
+    Array{T,N}(dims)
```

Comment:
or drop the brackets from the SharedArray one, that isn't julia syntax for optional arguments